sdk/release_notes.md

```diff
@@ -8,3 +8,11 @@
 
 - Re-add SDK refactors that were reverted in #2313 (#2347)
 - Address assembly scanning regression (#2347)
+- Updating to use `Microsoft.NET.Sdk.Functions.Generators` 1.2.2 (#2247)
+
+### Microsoft.Azure.Functions.Worker.Sdk.Generators 1.3.0
+
+- Introduces handling for `HttpResultAttribute`.
```

Comment:
Could we improve this to be a bit more specific please?

Reply:
Indeed. We want to have something that describes the feature here.